.circleci/config.yml

```diff
@@ -269,6 +269,32 @@ jobs:
       - store_test_results:
           path: target/reports/
 
+  itest-events-v2-provider:
```

Comment:
The job still does not run in the pipeline. It misses some dependency declarations at the bottom of the job (see workflows > main jobs).
👉 We should manually check that 1) `itest-cloudwatch-v2-provider` actually runs and 2) the test `test_events.TestEventsEventBus.test_create_custom_event_bus` passes 👀

Reply:
fixed in: https://github.com/localstack/localstack/pull/10553/commits/c584c7e79f4e5f86ddd466533fc586e7581ddee4 and https://github.com/localstack/localstack/pull/10553/commits/3cc18d4259434f57771751f235ba412a22dcadca

Reply:
great 👏 

Thanks for the clear traceability 💯